### PR TITLE
add slowfast model to video classification

### PR DESCRIPTION
**Pull request by huangjun12**
add slowfast model to video classification

### CODE REVIEW

File: dygraph/slowfast/train.py [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+        try:
+            from visualdl import LogWriter
+            vdl_writer = LogWriter(args.vd_logdir + '/' + str(local_rank))
+        except:
+            print(
+                "visualdl is not installed, please install visualdl if you want to use"
+            )
+
+    use_data_parallel = args.use_data_parallel
+    place = fluid.CUDAPlace(fluid.dygraph.parallel.Env().dev_id) \
```
Comment (chajchaj) [inline dygraph/slowfast/train.py]: can it suport cpu device？

File: dygraph/slowfast/train.py [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+            total_acc5 = 0.0
+            total_sample = 0
+
+            print('start for, Epoch {}/{} '.format(epoch,
+                                                   train_config.TRAIN.epoch))
+            batch_start = time.time()
+            for batch_id, data in enumerate(train_loader):
+                batch_reader_end = time.time()
+                y_data = data[2]
+                labels = to_variable(y_data)
```
Comment (chajchaj) [inline dygraph/slowfast/train.py:367]: to_variable之后，网络之前，建议添加个耗时统计，对于性能优化预计有较大帮助

File: dygraph/slowfast/eval.py [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        help='default use gpu.')
+    parser.add_argument(
+        '--use_data_parallel',
+        type=ast.literal_eval,
+        default=True,
+        help='default use data parallel.')
+    parser.add_argument(
+        '--weights',
+        type=str,
+        default="checkpoints/slowfast_epoch195",
```
Comment (shippingwang) [inline dygraph/slowfast/eval.py]: default is None?

File: dygraph/slowfast/train.py (hunk 1 above)
Comment (huangjun12) [reply dygraph/slowfast/train.py]: Done

File: dygraph/slowfast/eval.py (hunk 3 above)
Comment (huangjun12) [reply dygraph/slowfast/eval.py]: Done

**Review by shippingwang — APPROVED** [review]
LGTM

**Pull request merged**